### PR TITLE
Add note to README about major release

### PR DESCRIPTION
**Pull request by jeddy3**
> Which issue, if any, is this issue related to?

Ref: https://github.com/stylelint/stylelint/issues/5205#issuecomment-917683286

> Is there anything in the PR that needs further explanation?

Add a note before swapping the main branch to `v14`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+> This is the README for the [upcoming major release](CHANGELOG.md#Head). If you have time, please consider [contributing](CONTRIBUTING.md) to the [outstanding tasks](https://github.com/stylelint/stylelint/issues/5205).
```
Comment (ybiquitous) [inline README.md:1]: Removing a trailing whitespace is needed for Prettier 😉 

```suggestion
> This is the README for the [upcoming major release](CHANGELOG.md#Head). If you have time, please consider [contributing](CONTRIBUTING.md) to the [outstanding tasks](https://github.com/stylelint/stylelint/issues/5205).
```

**Review by ybiquitous — APPROVED** [review]
Thanks. LGTM 👍🏼

File: README.md (hunk 1 above)
Comment (jeddy3) [reply README.md:1]: Ha, that's what I get for cutting corners and using the GitHub interface to make the change... 😄

**Comment by jeddy3** [comment]
I've had a change of heart on this. Things like dependabot are working fine now, so don't want to have to revisit that just now.

**Pull request closed without merging**